### ISSUE TITLE
Aws::AutoScaling::Tag resource cannot correctly use #value method

### ISSUE THREAD

**Issue by ls-brentsmith** [aws/aws-sdk-ruby#1283]
I'm unable to create tags - because the initializer doesn't recognize the hash parameter :value as valid, and a tag cannot be created without it.

I also cannot use the method #value after creating tags manually through the console or via the Aws::AutoScaling::Client#create_or_update_tags. 

when using the resource 
```
[82] pry''> Aws::AutoScaling::Resource.new.group('FridayRC1-app')
=> #<Aws::AutoScaling::AutoScalingGroup name="FridayRC1-app">
[83] pry''> group = Aws::AutoScaling::Resource.new.group('FridayRC1-app')
=> #<Aws::AutoScaling::AutoScalingGroup name="FridayRC1-app">
[84] pry''> group = Aws::AutoScaling::Resource.new.group('FridayRC1-app')
=> #<Aws::AutoScaling::AutoScalingGroup name="FridayRC1-app">
[85] pry''> tag = group.tags.find {|t| t.key == 'test'}
=> #<Aws::AutoScaling::Tag key="test", resource_id="FridayRC1-app", resource_type="auto-scaling-group">
[86] pry''> tag.value
TypeError: no implicit conversion of Symbol into Integer
from /Users/ata/.chefdk/gem/ruby/2.1.0/gems/aws-sdk-resources-2.5.8/lib/aws-sdk-resources/resource.rb:223:in `[]'
```

Same group and tag via the client (which works fine, i'd just prefer to use the resource)

```
[87] pry''> client = Aws::AutoScaling::Client.new
=> #<Aws::AutoScaling::Client>
[88] pry''> resp = client.describe_auto_scaling_groups(auto_scaling_group_names: ['FridayRC1-app'])
=> #<struct Aws::AutoScaling::Types::AutoScalingGroupsType
[89] pry''> group = resp.auto_scaling_groups.first
=> #<struct Aws::AutoScaling::Types::AutoScalingGroup
[90] pry''> tag = group.tags.find {|t| t.key == 'test'}
=> #<struct Aws::AutoScaling::Types::TagDescription resource_id="FridayRC1-app", resource_type="auto-scaling-group", key="test", value="true", propagate_at_launch=true>
[91] pry''> tag.value
=> "true"
```

There is not a #create_tags method provided for the group resource, so I attempted to instantiate a tags resource and use the #create method, however it complains that I can't create one with a :value, while at the same time not allowing me to pass in a value in the options hash.

I would accept all of this as "just use the client instead" - but the documentation claims to have this method :)  http://docs.aws.amazon.com/sdkforruby/api/Aws/AutoScaling/Tag.html#value-instance_method

TL;DR - http://docs.aws.amazon.com/sdkforruby/api/Aws/AutoScaling/Tag.html#value-instance_method  always errors out.

**Comment by awood45** [comment]
This is a bug in the AutoScaling resource definition, which we've identified and I'm pushing up a fix (this bug would exist in all SDKs) to go out with the next release.
